dhis-2/dhis-support/dhis-support-external/src/test/java/org/hisp/dhis/external/conf/DhisConfigurationProviderTest.java

```diff
@@ -51,15 +70,50 @@ void isOn() {
 
   @Test
   void isEnabled() {
-    System.setProperty("dhis2.home", "src/test/resources");
-    DefaultLocationManager locationManager = DefaultLocationManager.getDefault();
-    locationManager.init();
-    DefaultDhisConfigurationProvider configProvider =
-        new DefaultDhisConfigurationProvider(locationManager);
-    configProvider.init();
     assertFalse(configProvider.isEnabled(ConfigurationKey.REDIS_ENABLED));
     assertFalse(configProvider.isEnabled(ConfigurationKey.MONITORING_API_ENABLED));
     assertTrue(configProvider.isEnabled(ConfigurationKey.ENABLE_QUERY_LOGGING));
     assertFalse(configProvider.isEnabled(ConfigurationKey.METHOD_QUERY_LOGGING_ENABLED));
   }
+
+  @Test
+  @DisplayName("remote servers retrieved from config should have expected values")
+  void getRemoteServersAllowedTest() {
+    // given there are 2 remote servers in the test config allowed list
+    // when we retrieve the remote servers allowed
+    List<String> remoteServersAllowed = configProvider.getRemoteServersAllowed();
+
+    // then it should contain the expected values
+    assertNotNull(remoteServersAllowed);
+    assertEquals(2, remoteServersAllowed.size());
+    assertTrue(
+        remoteServersAllowed.containsAll(
+            List.of("https://validtesturl.com/", "https://validtesturl2.com/")));
+  }
+
+  @ParameterizedTest
+  @NullSource
+  @ValueSource(strings = {"", " ", "https://invalidurl.com/fail"})
+  @DisplayName("invalid URLs should return false")
+  void invalidUrlTest(String url) {
+    // given there are 2 remote servers in the test config allowed list
+    // when we check if an invalid url is in the allowed list
+    boolean urlIsAllowed = configProvider.remoteServerIsInAllowedList(url);
+
+    // then it should be false
```

Comment:
Comments are good if they add any information but _it should be false_ is exactly what `assertFalse(urlIsAllowed);` already tells us. Nothing can be learned here so then it is just additional clutter to skip over. I suggest if you want to enhance this think about a message to add to the `assertTrue`/`assertFalse` because these in particular don't read very well without a message when checking booleans that do not simply reflect a boolean field somewhere.

Reply:
yep fair enough, it's just a habit when creating the skeleton of the test :)